plugins/slack_webhooks/app/decorators/stage_decorator.rb

```diff
@@ -6,6 +6,14 @@ def send_slack_webhook_notifications?
     slack_webhooks.any?
   end
 
+  def slack_buddy_channels
+    slack_webhooks.for_buddy.map(&:channel)
```

Comment:
> also memoize this since it will be used multiple times on 1 page ?

Yep, this is used twice on this page. The log shows that the SQL is hit twice:
```
  SlackWebhook Load (1.2ms)  SELECT "slack_webhooks".* FROM "slack_webhooks" WHERE "slack_webhooks"."stage_id" = ? AND "slack_webhooks"."for_buddy" = ?  [["stage_id", 2], ["for_buddy", "t"]]
  CACHE (0.2ms)  SELECT "slack_webhooks".* FROM "slack_webhooks" WHERE "slack_webhooks"."stage_id" = ? AND "slack_webhooks"."for_buddy" = ?  [["stage_id", 2], ["for_buddy", true]]
```
what's the best practice in memoizing this?